src/PiWeb.Import.Sdk/ImportFiles/IImportFile.cs

```diff
@@ -0,0 +1,70 @@
+﻿#region copyright
+
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+/* Carl Zeiss Industrielle Messtechnik GmbH        */
+/* Softwaresystem PiWeb                            */
+/* (c) Carl Zeiss 2024                             */
+/* * * * * * * * * * * * * * * * * * * * * * * * * */
+
+#endregion
+
+#region usings
+
+using System;
+using System.IO;
+using Zeiss.PiWeb.Import.Sdk.ImportFiles.Exceptions;
+
+#endregion
+
+namespace Zeiss.PiWeb.Import.Sdk.ImportFiles;
+
+/// <summary>
+/// Represents a file.
+/// </summary>
+public interface IImportFile
+{
+    #region properties
+
+    /// <summary>
+    /// The name of this file including the extension.
+    /// </summary>
+    public string Name { get; }
```

Comment:
Could you unify the usage of modifiers here?
I assume the latest decision was to drop them in interfaces.

Reply:
Done.